%YAML:1.0
---
calibrationTimeChar: "Sun May 26 16:31:00 2024\n"
calibrationTime_t: 1716712260
imageWidth: 1280
imageHeight: 1024
cameraMatrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 8.8136100819112039e+02, 0., 6.4220272461712216e+02, 0.,
       8.8058628963515389e+02, 5.0584011804970760e+02, 0., 0., 1. ]
disCoffes: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ -9.3578985363918982e-02, 7.3018505781808796e-02,
       -3.8644741900791793e-04, -9.8636591357284249e-05,
       1.4049603863372971e-02 ]
newCameraMatrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 8.5681836548890794e+02, 0., 6.4135126096624094e+02, 0.,
       8.5612924539197581e+02, 5.0472873206422679e+02, 0., 0., 1. ]
newDisCoffes: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ -1.8139423750434592e-04, 8.0422815548505239e-04,
       -3.5311314814390710e-05, -4.3912675602818084e-05,
       -1.1367697881842806e-03 ]
